File: Prova 1/1045_tiposTriangulos.c
#include <stdio.h>
#include <math.h>

int main() {
	double a, b, c, maior, meio, menor;
	scanf("%lf %lf %lf", &a, &b, &c);
	
	if (a >= b && a >= c) {
		maior = a;
		meio =  b;
		menor = c;
	}
	
	if (b >= a && b >= c) {
		maior = b;
		meio = a;
		menor = c;
	} else {
		maior = c;
		meio = a;
		menor = b;
	}
	
	if (maior >= meio + menor)
		printf("NAO FORMA TRIANGULO\n");
	else if (maior * maior == meio * meio + menor * menor)
		printf("TRIANGULO RETANGULO\n");
	else if (maior * maior > meio * meio + menor * menor)
		printf("TRIANGULO OBTUSANGULO\n");
	else if (maior * maior < meio * meio + menor * menor)
		printf("TRIANGULO ACUTANGULO\n");
	
	if (maior == menor && maior == meio && meio == maior && meio == menor && menor == maior && menor == meio)
		printf("TRIANGULO EQUILATERO\n");
	else if (maior == menor|| maior == meio || meio == maior || meio == menor || menor == maior || menor == meio)
		printf("TRIANGULO ISOSCELES\n");
}

File: Prova 2/1045_tiposTriangulos.py
# lendo a, b e c em uma linha usando split()

a, b, c = input().split()

#convertendo para float

a = float(a)
b = float(b)
c = float(c)


if ((a >= b) and  (a >= c)):
	maior = a;
	meio = b;
	menor = c;

if ((b >= a) and (b >= c)):
	maior = b
	meio = a
	menor = c
else:
	maior = c
	meio = a
	menor = b

	
if (maior >= meio + menor):
	print("NAO FORMA TRIANGULO")
elif (maior * maior == meio * meio + menor * menor):
	print("TRIANGULO RETANGULO")
elif (maior * maior > meio * meio + menor * menor):
	print("TRIANGULO OBTUSANGULO")
elif (maior * maior < meio * meio + menor * menor):
	print("TRIANGULO ACUTANGULO")
		
if (maior == menor and maior == meio and meio == maior and meio == menor and menor == maior and menor == meio):
	print("TRIANGULO EQUILATERO")
elif (maior == menor or maior == meio or meio == maior or meio == menor or menor == maior or menor == meio):
	print("TRIANGULO ISOSCELES")
